tests/typings/sync/sync.ts

```diff
@@ -128,31 +128,31 @@ const el3 = el2.$('')
 el1.getCSSProperty('style')
 el2.click()
 el1.moveTo({ xOffset: 0, yOffset: 0 })
-expectType<true | void>(
+expectType<Promise<boolean>>(
```

Comment:
Why is this `Promise<boolean>` it should behave the same as `waitForDisplayed` and others.

Reply:
I think `async` is missing for waitForExist

Current declaration is 
```export default function waitForExist``` in https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/element/waitForExist.ts

so, It is expecting a Promise<boolean>

Reply:
Also am using node 16. May be an issue is due to this, I will check with node 14(sync mode dependency) too!

Reply:
Am seeing the same issue with node 12 and 14. May be an issue in `waitForExist` declaration

```
sync.ts:132:5 - error TS2345: Argument of type 'Promise<boolean>' is not assignable to parameter of type 'boolean'.
132     el2.waitForExist({
133         timeout: 1,
... 
136         reverse: true
137     })
```
Not sure how it worked previously then!
Any suggestions @christian-bromann !